src/js/lib/monster.ui.js

```diff
@@ -3134,7 +3134,10 @@ define(function(require) {
 		attributes.class = _
 			.chain(attributes)
 			.get('class', '')
-			.split(' ')
+			.split(/\s+/g)	// Split by one or more whitespaces
+			.filter(function(className) {
```

Comment:
Nice addition to trim extra empty spaces and check against empty strings.

We could use a declarative style to make it more concise AND understandable!

To do that, you can do the opposite operation: reject empty strings instead of keeping non empty ones.
```javascript
_.reject(['my-css-class', ''], _.isEmpty)
```
This should give you the exact same output at the end of you chain.

Reply:
:open_mouth: nice! It's shorter. I'll use that. thanks :+1: